common/src/Model/Entity.cpp

```diff
@@ -226,6 +226,15 @@ namespace TrenchBroom {
             visitor.visit(this);
         }
 
+        NodeList Entity::nodesRequiredForViewSelection() const {
+            if (hasChildren()) {
+                // Selecting a brush entity means selecting the children
+                return children();
+            } else {
+                return NodeList{const_cast<Entity*>(this)};
```

Comment:
That's evil. I'd rather remove the const modifier from the method.

Reply:
Yeah, agreed, will do that. 
Various other Node functions that are `const` have the same issue.. children(), parent(), findDescendants(), although it's less blatantly wrong because they don't directly return `this`.